internal/controller/model_controller.go

```diff
@@ -68,10 +68,23 @@ func (r *ModelReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl
 func (r *ModelReconciler) reconcileModel(ctx context.Context, model *apiv1.Model) (result, error) {
 	log := log.FromContext(ctx)
 
-	if model.Status.Artifacts.URL != "" {
+	if model.Status.Ready {
```

Comment:
Keep this change to the conditional

Reply:
you mean change model.Status.Ready == true?

Reply:
I think what I have currently is preferred over having
```
if model.Status.Ready == true {
```

Source: https://stackoverflow.com/questions/3685002/check-if-boolean-is-true

Reply:
Was just commented on checking ready instead of the url

Reply:
not sure why I misunderstood your comment. I read it as change to the conditional format or smth like that.